README.md

```diff
@@ -3,6 +3,8 @@
 Several tests are available for the user to benchmark Pyccel against other common accelerators, notably [pythran](https://pythran.readthedocs.io/en/latest/) and [numba](https://numba.pydata.org/).
 The same code is used for all tests, only the decorators change.
 
+The dependencies can be installed using the command `python3 -m pip install .
+
 The code can be executed by running the script `run_benchmarks.py`.
```

Comment:
Here and on line 11 of README.md, the path to `run_benchmarks.py` is not updated to reflect that `run_benchmarks.py` was moved to `benchmarks/`.